apps/condo/pages/_app.tsx

```diff
@@ -85,15 +104,18 @@ import {
 } from '@condo/domains/ticket/hooks/useTicketDocumentGenerationTaskUIInterface'
 import { useTicketExportTaskUIInterface } from '@condo/domains/ticket/hooks/useTicketExportTaskUIInterface'
 import { CookieAgreement } from '@condo/domains/user/components/CookieAgreement'
-import { USER_QUERY } from '@condo/domains/user/gql'
+
+import Error404Page from './404'
```

Comment:
You can try to utilise next's `_error` page instead

Reply:
this is a bit problematic and will not work as you expect